uai2017/process/pc_responseInvitationProcess_uai2017.js

```diff
@@ -0,0 +1,34 @@
+function() {
+
+  var or3client = lib.or3client;
+  var hashKey = or3client.createHash(note.content.email, "2810398440804348173");
+  if(hashKey == note.content.key) {
+    if (note.content.response == 'Yes') {
+      console.log("Invitation replied Yes")
+      //if a user is in the declined group, remove them from that group and add them to the reviewers group
+      or3client.removeGroupMember('auai.org/UAI/2017/Program_Committee/declined', note.content.email, token)
+      or3client.addGroupMember('auai.org/UAI/2017/Program_Committee', note.content.email, token)
```

Comment:
I guess, is it the removeGroupMember promise executed? don't you need to call .then ?